tests/unit/show_tuner_costs.c

```diff
@@ -19,43 +19,49 @@ void dummy_logger(ncclDebugLogLevel level, unsigned long flags, const char *file
 
 int main(int argc, const char **argv)
 {
-	printf("algo;proto;nodes;ranks;size;cost\n");
+	printf("algo;proto;nodes;ranks;channels;size;cost\n");
+	// clang-format off
 	for (int algo = 0; algo < 6; ++algo) {
-		for (int proto = 0; proto < NCCL_NUM_PROTOCOLS; ++proto) {
-			for (size_t nodes = 1; nodes <= 512; nodes <<= 1) {
-				for (size_t ranks_per_node = 1; ranks_per_node <= 8; ranks_per_node <<= 1) {
-					struct nccl_ofi_tuner_context *context = NULL;
-					if (ncclTunerPlugin_v2.init(ranks_per_node * nodes,
-								    nodes,
-								    dummy_logger,
-								    (void**)&context) != 0) {
-						return 1;
-					}
+	for (int proto = 0; proto < NCCL_NUM_PROTOCOLS; ++proto) {
+	for (size_t nchan = 2; nchan <= 32; nchan <<= 1) {
```

Comment:
i know there's no win here, and it's a unit test, but indenting looks funky.  I'm fine if you leave it.

Reply:
this was intentional to prevent the lines from getting so absurdly wrong that the wrapping became ridiculous. We could move the variables outside of the loop, but I think this is an ok middleground.

Reply:
yeah, as long as it's intentional, go with it.